CMakeLists.txt

```diff
@@ -20,15 +20,19 @@ endif()
 project(serac LANGUAGES CXX C)
 
 # MPI is required in Serac.
-set(ENABLE_MPI ON CACHE BOOL "")
-
-if (NOT MPI_C_COMPILER OR NOT MPI_CXX_COMPILER)
-    message(FATAL_ERROR 
-            "Serac requires MPI.  It is required to provide the MPI C/C++ "
-            "compiler wrappers via the CMake variables, "
-            "MPI_C_COMPILER and MPI_CXX_COMPILER.")
+# Note: for the style only mode, which doesn't need to actually build, we need to ignore this
+if (NOT SERAC_STYLE_ONLY)
+    set(ENABLE_MPI ON CACHE BOOL "")
```

Comment:
Is this to save time or did an error occur when ENABLE_MPI was left on?

Reply:
Yes I ran into several issues with the base container and it just wasn't worth figuring out since all i want is the `style` build target.